ports/sdlpal/port.json

```diff
@@ -13,7 +13,7 @@
     ],
     "desc": "The Legend of Sword and Fairy, is a Chinese language fantasy video game series and media franchise centered on a series of nine Chinese mythology/xianxia-themed adventure role-playing computer games created by Yao Zhuangxian.​",
     "desc_md": "The Legend of Sword and Fairy (Chinese: 仙劍奇俠傳; pinyin: Xiānjiàn Qíxiá Zhuàn), also known as Sword and Fairy (仙劍) or Chinese Paladin, is a Chinese language fantasy video game series and media franchise centered on a series of nine Chinese mythology/xianxia-themed adventure role-playing computer games created by Yao Zhuangxian. The eponymous first game in the series is released for MS-DOS CLI in 1995 by the 'Crazy Boyz' team (狂徒創作群) of the Taiwanese game developing and publishing company Softstar Entertainment (大宇資訊). It is one of the so-called 'Twin Swords of Softstar' (大宇雙劍) along with the sister Xuan-Yuan Sword series, and has been widely regarded as one of the (if not the) most iconic Chinese RPG series ever made.​",
-    "inst": "Purchase the game from Steam: https://store.steampowered.com/app/1546570/Sword_and_Fairy/ and put all files into the sdlpal/gamedata folder",
+    "inst": "Purchase the game from Steam: https://store.steampowered.com/app/1546570/Sword_and_Fairy/ and put all files into the sdlpal/gamedata folder. Add the english translation into the sdlpal/gamedata from: https://www.romhacking.net/translations/2441/. If you want to play in other language, modify the variable 'MessageFileName' at the file sdlpal/sdlpal.cfg with the one you want or empty for original language.",
```

Comment:
Would it be better for the user experience if the launchscript could detect a translation file and add a sed to the configfile?